daemon/process/CMakeLists.txt

```diff
@@ -108,8 +108,9 @@ install( # Specifies the target to install
 
 
 # Install the config file for the target platform
-if( RDK_PLATFORM STREQUAL "XI6" )
-        set( DOBBY_SETTINGS_FILE "settings/dobby.xi6.json" )
+# Generic is just a standard build with no extra settings
```

Comment:
I think we should keep the xi6 specific configuration even though it's not used in current builds.